doc/source/notebooks/advanced/mcmc.pct.py

```diff
@@ -589,3 +598,94 @@ def run_chain_fn():
     ax.hist(parameter_samples[name_to_index[hyp]], bins=20)
     ax.set_title(hyp)
 plt.tight_layout()
+
+# %% [markdown]
+# ## Prior on constraint and unconstrained parameters
+
+# %% [markdown]
+# GPflow's :class:`Parameter` class provides options for setting a prior. :class:`Parameter` wraps a constrained tensors and
```

Comment:
Have you checked that nbsphinx actually converts the `:class:` into a proper link? If it still says ":class:" in the docs that will look pretty weird..

Reply:
https://sphinx-rtd-tutorial.readthedocs.io/en/latest/docstrings.html#an-example-class-with-docstrings